tests/core/contracts/test_extracting_event_data.py

```diff
@@ -31,8 +31,8 @@ def Emitter(web3, EMITTER):
 @pytest.fixture()
 def emitter(web3, Emitter, wait_for_transaction, wait_for_block, address_conversion_func):
     wait_for_block(web3)
-    deploy_txn_hash = Emitter.constructor().transact({'from': web3.eth.coinbase, 'gas': 1000000})
-    deploy_receipt = web3.eth.wait_for_transaction_receipt(deploy_txn_hash)
+    deploy_txn_hash = Emitter.constructor().transact({'gas': 10000000})
+    deploy_receipt = wait_for_transaction(web3, deploy_txn_hash)
```

Comment:
These changes are a bit cosmetic but these fixtures are copied about 5 times or so across the codebase and I just changed them all to be the same for consistency. Leaves out questions and now it actually uses the `wait_for_transaction` fixture that is being passed in here 😉